src/modules/item/utils.ts

```diff
@@ -107,7 +107,9 @@ export function getMetadata(item: Item) {
     case ItemType.WEARABLE: {
       const data = item.data as WearableData
       const bodyShapes = getBodyShapes(item)
-      return `${version}:${type}:${slug}:${data.category}:${bodyShapes.join(',')}`
+      return `${version}:${type}:${item.name}:${item.description}:${data.category}:${bodyShapes
+        .map(bodyShape => bodyShape.split('/').pop())
```

Comment:
I'm wondering if we can add a comment here explaining why we are doing 
```javascript
bodyShape.split('/').pop()
```

Like: 
```bash
// bodyshapes are dcl://BaseFemale and dcl://BaseMale. We store only the name of the shape: BaseFemale or BaseMale
```

Reply:
done 👍